library_generation/generate_showcase_integration_test.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+set -e
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+source $SCRIPT_DIR/utilities.sh
+SDK_WORKSPACE=$SCRIPT_DIR/../WORKSPACE
+GAX_PROPERTIES=$SCRIPT_DIR/../gax-java/dependencies.properties
+SHOWCASE_BUILD_FILE=$SCRIPT_DIR/../showcase/BUILD.bazel
+
+cd $SCRIPT_DIR
+rm -rdf output
+
+# clone gapic-showcase
+if [ ! -d schema ]; then
+  git clone https://github.com/googleapis/gapic-showcase
+  pushd gapic-showcase
+  showcase_version=$(get_version_from_WORKSPACE "_showcase_version" $SDK_WORKSPACE)
+  git checkout "v$showcase_version"
+  mv schema ..
+  popd
+  rm -rdf gapic-showcase
+fi
+if [ ! -d google ];then
+  git clone https://github.com/googleapis/googleapis
+  mv googleapis/google .
+  mv googleapis/WORKSPACE ..
+  rm -rdf googleapis
+fi
+
+GOOGLEAPIS_WORKSPACE=$SCRIPT_DIR/WORKSPACE
+
+ggj_version=$(get_version_from_WORKSPACE _gapic_generator_java_version $SDK_WORKSPACE)
+if [ $(echo $ggj_version | grep 'SNAPSHOT' | wc -l) -gt 0 ]; then
+  echo 'This repo is at a snapshot version. Installing locally...'
+  pushd $SCRIPT_DIR/..
+  mvn clean install -DskipTests -Dclirr.skip
+  popd
+fi
+
+
+grpc_version=$(get_version_from_properties "version.io_grpc" $GAX_PROPERTIES)
```

Comment:
Why don't we get grpc version from `gapic-generator-java-pom-parent/pom.xml`?

Reply:
Good point. I just saw the [convenience method](https://github.com/googleapis/sdk-platform-java/blob/880c27ba4f4e3efbf87dc11ee9454bd762158197/library_generation/generate_library.sh#L62) to infer from the parent pom. I will rely on this instead.

Reply:
Omitted grpc version in showcase script